### DISCUSSION TITLE
Slideshow has problems loading images after running for a while

### DISCUSSION THREAD

**Discussion by midiwidi** [immich-app/immich#9340] [Q&A]
I made an album containing all my images and videos (>5000). On my tablet I run a slideshow of this album (shuffled order) in the Fully Kiosk Browser to mimic a photo frame.
This works well but after running for many hours the images don't load anymore. I only get a black screen with a spinner indicating that the image is loading. Interestingly the videos still work. Every now and then when the slideshow gets to a video it is played but after that it goes back to a black screen with a spinner trying to load the next image. 
Before I report that as a bug I thought I'll collect more information. Is there a log file on my immich server I could monitor? I don't think there would be anything on the client (my tablet) I could look at since this is running in a browser.

**Comment by bo0tzz** [comment]
My first hunch would be a memory leak on the client, but it's worth seeing if there's any error logs on the server anyways (https://immich.app/docs/guides/docker-help#logs).

Can you reproduce this on a PC? If so it'd be easier to get some insights via the browser devtools.

**Reply by midiwidi** [reply]
I'll have a look at the logs and start the slideshow in Firefox on my server. That would be on the same machine as the one immich runs on but better than nothing.
I'll report here if I find anything out.

**Comment by midiwidi** [comment]
I think what might have happen is that the tablet lost network connection for a moment and if that happens at the right time the slideshow doesn't continue even if the network connection is back. After I found that out on my tablet I tried to reproduce this on my PC while having the developer tools open in Firefox. I turned off Wifi for a brief moment and while the slideshow often continues after the connection is back, sometimes it doesn't and shows the black screen with the spinner I have seen on my tablet. Here is a screenshot with the network tab and the log window open in the developer tools.

![grafik](https://github.com/immich-app/immich/assets/54610151/fb1fc258-985a-4db3-9a22-0b9d081e2a42)

When it is in this state I can get the slideshow to continue by clicking the "next image" icon in the overlayed controls.

**Comment by midiwidi** [comment]
I changed from the Fully Kiosk Browser to Chrome for Android and to the Kiwi Browser which is based on Chrome.  Both show the identical behaviour to the Fully Kiosk Browser. After the Immich slide show was running for a while (sometimes it only takes some hours but at other times it takes over a day) it stops showing images and I get just a black screen with a spinner. Interestingly, if the current content is a video then it is shown but not images. I connected the tablet via USB and was able to remotely debug Chrome / Kiwi in it from my PC.
Here is a screenshot
![grafik](https://github.com/immich-app/immich/assets/54610151/c54e61ec-88f3-49f0-a8c6-c8cb79fc2935)
The network traffic tab shows the attempt to pull images every 10s. The long bar in the waterfall window is from when a video was shown instead of images.
I have saved the traffic from the network tab as a *.har file which you can investigate using Chrome.
[immich_slideshow_kiwi_errorstate.zip](https://github.com/user-attachments/files/15526030/immich_slideshow_kiwi_errorstate.zip)
I also saved the messages from the console window
[immich_slideshow_kiwi_errorstate_console.log](https://github.com/user-attachments/files/15526047/immich_slideshow_kiwi_errorstate_console.log)
I hope this helps to identify what the problem is. Please let me know if you need more information.

**Reply by midiwidi** [reply]
Should I repost that as an actual issue?

**Reply by alextran1502** [reply]
Yes please, and we might have a fix in #9883

**Comment by midiwidi** [comment]
I have opened issue #9991 for the problem described above. Please comment there.
